helpdesk/templates/tickets/ticket_detail.html

```diff
@@ -75,31 +95,23 @@
         </article>
         {% endfor %}
 
-        {% if object.resolution %}
-            <article class="message is-success">
-                <div class="message-header">
-                    <p>Resolved at {{ object.resolution.resolved_at }} by {{ object.resolution.user }}</p>
-                </div>
-                {% if object.resolution.comment %}
-                <div class="message-body">
-                    {{object.resolution.comment|linebreaks}}
-                </div>
-                {% endif %}
-            </article>
-        {% else %}
         <article class="message is-info">
             <div class="message-body">
                 <form method="post">
                     {% csrf_token %}
                     {{ comment_form | crispy }}
                     <div class="control">
                       <button class="button is-info" type="submit" formaction="{% url 'tickets:ticket_comment' object.id %}">Comment</button>
+                      {% if object.status != "RS" %}
                       <button class="button is-warning" type="submit" formaction="{% url 'tickets:ticket_resolve' object.id %}">Comment and Resolve</button>
+                      {% endif %}
+                      {% if object.status != "OP" %}
+                      <button class="button is-success" type="submit" formaction="{% url 'tickets:ticket_reopen' object.id %}">Comment and Re-Open</button>
+                      {% endif %}
```

Comment:
Would be nice to loop over statuses here in the future? e.g Escalate and comment